e2e/_suites/ingest-manager/stand-alone.go

```diff
@@ -131,6 +147,39 @@ func (sats *StandAloneTestSuite) getContainerLogs() error {
 	return nil
 }
 
+// installTestTools we need the container name because we use the Docker Client instead of Docker Compose
+// we are going to install those tools we use in the test framework for checking
+// and verifications
+func (sats *StandAloneTestSuite) installTestTools(containerName string) error {
+	if sats.Image != "ubi8" {
+		return nil
+	}
+
+	cmd := []string{"microdnf", "install", "procps-ng"}
```

Comment:
@jsoriano we had to do this to be able to check for process existence in the test container.

I think it won't cause any overhead

Reply:
Yeah, I don't think it will cause any issue. How are the checks we do? Have we alternatively considered reading the procfs directly? (something like `readlink /proc/*/exe | grep /metricbeat` might work).

Reply:
we are running `pgrep -n -l $PROCCESS`, with PROCESS=[elastic-agent | filebeat | metricbeat]

Reply:
The thing is, we are using the docker client to send the command, so any pipe `|` could be interpolated in the host instead of in the container

Reply:
Oh ok, I see. Well, the "grep" could be done in the client side, as we check now with `strings.Contains` after `pgrep`.

Reply:
I'm gonna give it a try, although not sure about the glob expression with the star. Will post results, thanks!

Reply:
Running that command from inside the container works:

```shell
docker exec -ti ingest-manager_elastic-agent_1 bash
[root@d571995e1f80 elastic-agent]# readlink /proc/*/exe
/usr/share/elastic-agent/data/elastic-agent-b595bf/elastic-agent
/usr/share/elastic-agent/data/elastic-agent-b595bf/install/metricbeat-8.0.0-SNAPSHOT-linux-x86_64/metricbeat
/usr/share/elastic-agent/data/elastic-agent-b595bf/install/filebeat-8.0.0-SNAPSHOT-linux-x86_64/filebeat
/usr/share/elastic-agent/data/elastic-agent-b595bf/install/metricbeat-8.0.0-SNAPSHOT-linux-x86_64/metricbeat
/usr/bin/bash
/usr/bin/readlink
/usr/bin/readlink
```

But running it from outside, does not:
```shell
$ docker exec ingest-manager_elastic-agent_1 readlink /proc/*/exe
zsh: no matches found: /proc/*/exe
$ docker exec ingest-manager_elastic-agent_1 readlink "/proc/*/exe"
$ docker exec  ingest-manager_elastic-agent_1 "readlink /proc/*/exe"
OCI runtime exec failed: exec failed: container_linux.go:349: starting container process caused "exec: \"readlink /proc/*/exe\": stat readlink /proc/*/exe: no such file or directory": unknown
```

Reply:
Interesting, it seems that `zsh` is trying to resolve the wildcard in the host. This should work no matter the shell in the host:
```
$ docker exec ingest-manager_elastic-agent_1 sh -c 'readlink /proc/*/exe'
```
Or directly:
```
$ docker exec ingest-manager_elastic-agent_1 sh -c 'grep /metricbeat /proc/*/exe'
```
And check exit status.

Reply:
But don't spend a lot of time with this, installing tools if needed looks good too :slightly_smiling_face: